app/views/session.py

```diff
@@ -69,8 +70,8 @@ def login():
     cookie_session['theme'] = g.schema.json['theme']
     cookie_session['survey_title'] = g.schema.json['title']
 
-    if 'account_service_url' in claims and claims.get('account_service_url'):
-        cookie_session[ACCOUNT_URL] = claims.get('account_service_url')
+    if 'account_url' in claims and claims.get('account_url'):
```

Comment:
I know its just renamed from previous but we don't need the first check so can we remove it?. So just `if claims.get('account_url'):`